test/test_trixi.jl

```diff
@@ -155,7 +155,10 @@ macro test_nowarn_mod(expr, additional_ignore_content = String[])
                                      # TODO: Silence warning introduced by Flux v0.13.13. Should be properly fixed.
                                      r"┌ Warning: Layer with Float32 parameters got Float64 input.+\n│.+\n│.+\n│.+\n└ @ Flux.+\n",
                                      # NOTE: These warnings arose from Julia 1.10 onwards
-                                     r"WARNING: Method definition .* in module .* at .* overwritten .*.\n"]
+                                     r"WARNING: Method definition .* in module .* at .* overwritten .*.\n",
+                                     # Warnings from third party packages
+                                     r"┌ Warning: Problem status ALMOST_INFEASIBLE; solution may be inaccurate.\n└ @ Convex ~/.julia/packages/Convex/.*\n",
+                                     r"┌ Warning: Problem status ALMOST_OPTIMAL; solution may be inaccurate.\n└ @ Convex ~/.julia/packages/Convex/.*\n"]
```

Comment:
Is there no way to disable the warnings programmatically in the extension? If not, feel free to close this convo

Reply:
Hm, I guess the warnings are useful to the user but should not prohibit CI from passing - that is why there are added here.

Reply:
Ok, fair enough! /r